src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -95,6 +96,35 @@ public void claimantDefaults(CaseData caseData) {
         } else {
             caseData.setClaimant(MISSING_CLAIMANT);
         }
+        addClaimantDocuments(caseData);
+    }
+
+    private void addClaimantDocuments(CaseData caseData) {
```

Comment:
I think this method could be cut down. Rather than going through the docCollection multiple times for each doc type, you could have a list of doc types you want and then have a .contains

```
    private List<String> submissionDocs = List.of(ET1_DOC_TYPE, ACAS_DOC_TYPE, ET1_ATTACHMENT_DOC_TYPE);

    private void addClaimantDocs(CaseData caseData) {
        if (CollectionUtils.isNotEmpty(caseData.getDocumentCollection())) {
            List<DocumentTypeItem> claimantDocs = caseData.getDocumentCollection().stream()
                    .filter(d -> submissionDocs.contains(d.getValue().getTypeOfDocument()))
                    .toList();
            caseData.setClaimantDocumentCollection(claimantDocs);
        }
    }
```

Reply:
yes, good idea but I think if we have a list of doc types and use 'contains', we still have a collection in which we will have 3 docs and we still need to get different docs (et1, attachment, acas).

Reply:
refactored code now, I think that's a good idea